staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache_patch.go

```diff
@@ -0,0 +1,63 @@
+package cacher
```

Comment:
patch_watch_cache.go is our pattern, isn't it?

Reply:
I think it is correct, we have `store_patch.go`, here:
https://github.com/kcp-dev/kubernetes/blob/a8b2000cd5878fb3d0b8dbc74bec068c0a034a8e/staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_patch.go